config/render/capm3.yaml

```diff
@@ -2618,16 +2562,6 @@ spec:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8085/
-        - --logtostderr=true
-        - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.16.0
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
```

Comment:
Where are the RBAC roles now required for controller runtime?
I think you forgot those.

Reply:
Oh! @kashifest we need to test this and probably add some new RBAC

Reply:
Indeed! I have put a PR for that https://github.com/metal3-io/baremetal-operator/pull/2116, I haven't tested them yet but those seem to me the missing RBACs